datadog_checks_dev/datadog_checks/dev/tooling/commands/agent/changelog.py

```diff
@@ -61,6 +69,9 @@ def changelog(since, to, write, force):
                 else:
                     display_name = name
 
+                if display_name == 'Mesos':
```

Comment:
should we also make this a constant ie `display_name_mappings`?

Reply:
I guess we can, but `mesos_slave` is very unique due to how we migrated it, so it should be the only one affected